target/linux/x86/base-files/etc/board.d/02_network

```diff
@@ -45,28 +45,78 @@ roqos-roqos-core-rc10)
 	;;
 sophos-sg-105r1|sophos-xg-105r1| \
 sophos-sg-105wr1|sophos-xg-105wr1| \
-sophos-sg-105r2|sophos-xg-105r2| \
-sophos-sg-105wr2|sophos-xg-105wr2| \
 sophos-sg-115r1|sophos-xg-115r1| \
 sophos-sg-115wr1|sophos-xg-115wr1| \
-sophos-sg-115r2|sophos-xg-115r2| \
-sophos-sg-115wr2|sophos-xg-115wr2| \
 sophos-xg-85*|sophos-xg-86*)
 	ucidef_set_interfaces_lan_wan "eth0 eth2 eth3" "eth1"
 	;;
 sophos-sg-125r1|sophos-xg-125r1| \
 sophos-sg-125wr1|sophos-xg-125wr1| \
-sophos-sg-125r2|sophos-xg-125r2| \
-sophos-sg-125wr2|sophos-xg-125wr2| \
 sophos-sg-135r1|sophos-xg-135r1| \
-sophos-sg-135wr1|sophos-xg-135wr1| \
-sophos-sg-135r2|sophos-xg-135r2| \
-sophos-sg-135wr2|sophos-xg-135wr2)
+sophos-sg-135wr1|sophos-xg-135wr1)
 	ucidef_set_interfaces_lan_wan "eth0 eth2 eth3 eth4 eth5 eth6 eth7" "eth1"
 	;;
-sophos-sg-135r3|sophos-xg-135r3| \
-sophos-sg-135wr3|sophos-xg-135wr3)
-	ucidef_set_interfaces_lan_wan "eth0 eth1 eth2 eth3 eth5 eth7 eth8" "eth6"
+sophos-sg-105r2|sophos-sg-105wr2| \
+sophos-sg-115r2|sophos-sg-115wr2)
+	ucidef_set_network_device_path "eth0" "pci0000:00/0000:00:1c.0/0000:01:00.0"
+	ucidef_set_network_device_path "wan"  "pci0000:00/0000:00:1c.1/0000:02:00.0"
```

Comment:
wasn't this 'eth1' instead of 'wan' earlier? Looking at a picture of a Sophos-SG-105W (i.e. from here:
https://utm-shop.de/firewall/sophos/sophos-sg/sg105/hardware-appliance/1520/sophos-sg-105w-security-appliance-wifi-sg105w?c=3726 - no this isn't an endorsement) 

it looks like the front-panel and back there are always two labels on each port. Eth0-Eth3 is written above and the usage below (lan, wan, dmz, ha)

though, I have never used Sophos devices, or flexible are those mappings?

Reply:
Is there an issue marking one port `wan` instead of `eth1`? I believe `wan` would be a better visual for things like port status in luci, for example:
![image](https://github.com/openwrt/openwrt/assets/4768176/23696a15-6aa7-4abb-9654-804be57d0834)

Reply:
> Is there an issue marking one port wan instead of eth1? I believe wan would be a better visual for things like port status in luci.

I think that you can apply that argument ("better visual ...") for the other ports too! So why WAN specifically and not make eth2=DMZ, eth3=HA and eth1=as just the only LAN?

Won't these be nice to have too? (But then I also have an example for going with eth1-4. Because on the other hand, here at the Institute there's a blind lady. Saying to her to connect something to a "HA"-Port makes no sense, but she would understand that it's the fourth port from the right... not that she dabbles with the network setup).

Now getting, HA (high availability / should this just be bonding?) and DMZ (different vendors have slightly different ideas about DMZ) work "out of the box" with what ucidef provides out of the box is though.

Reply:
> I think that you can apply that argument ("better visual ...") for the other ports too! So why WAN specifically and not make eth2=DMZ, eth3=HA and eth1=as just the only LAN?
> Won't these be nice to have too? 

I've considered this. But WAN is actually WAN, all the other ports are just LAN in OpenWrt default configuration. I think that marking a port DMZ while it doesn't add any DMZ-related functionality is misleading.

Is there an issue marking one port `wan` instead of `eth1`?

If someone wants to take this another step further and add more code to support actual LAN/HA/DMZ functionality on different ports (and rename them), at least this gives the foundation with the sysfs paths. However, I believe most people would use these devices with just a single WAN port and multiple LAN ports, so I'm not sure if such a development would be welcome in a PR.

Reply:
> I've considered this. But WAN is actually WAN, all the other ports are just LAN in OpenWrt default configuration. I think that marking a port DMZ while it doesn't add any DMZ-related functionality is misleading.
>Is there an issue marking one port wan instead of eth1?

Please excuse me here. But what do you mean with "is there an issue", when you are describing the inconsistency with the port naming in the very same post?

Ideally, the OpenWrt ports should try to match the printed function on the front/backpanel for the specific port. Furthermore, I fully understand that  DMZ and HA are not implemented with ucidef_* ... but your are contradicting with your PR message states "[...] this commit addresses that and assigns proper ports to match the markings on the case." ... You lump together these HA/DMZ as part of the lan bridge... 

However, what is nice about the commit is that you are assigning the right eth0/1-eth4/8 labels with what the port is labeled on the front/back. So I would concentrate on that.

Reply:
> but your are contradicting with your PR message states "[...] this commit addresses that and assigns proper ports to match the markings on the case." ... You lump together these HA/DMZ as part of the lan bridge...

No, I do not lump them together, they were already "lumped together" previously, so I'm keeping the status quo and I'm not creating misleading titles.

This commit, however for some models changes the network file from:
```
config device
	option name 'br-lan'
	option type 'bridge'
	list ports 'eth0'
	list ports 'eth1'
	list ports 'eth2'
	list ports 'eth3'
	list ports 'eth5'
	list ports 'eth7'
	list ports 'eth8'
```
Where the ethX ports DID NOT correspond to the actual markings on the case to:
```
config device
	option name 'br-lan'
	option type 'bridge'
	list ports 'eth1'
	list ports 'eth3'
	list ports 'eth4'
	list ports 'eth5'
	list ports 'eth6'
	list ports 'eth7'
	list ports 'eth8'
```
Where the ethX ports DO correspond to actual markings on the case.

This commit also takes into consideration that the ports on SG and XG lines are marked differently, for example on the SG models the ports are marked eth0-eth3, whereas on the XG models the ports are marked eth1-eth4.

So overall, this commit does assign proper ports to reflect **ethX** markings on the case, with exception of WAN, which is marked WAN.

Reply:
I agree to name the WAN port wan and not ethX.
We already put the WAN port into the WAN firewall zone and all other ports into the LAN zone.

This does not rename the Linux network interfaces, is that correct?
Is the eth4 port in UCI now different from the eth4 interface in Linux? This could confuse people.

Reply:
@hauke -- now, as in before this patch is applied, the eth1 port as marked on XG-135 case for example is eth7 port in OpenWrt. I believe the 4-port devices are not affected by the port numbering as badly as the 8-port devices, but with 8-port devices this aims to fix the misnumbered ports. For 4-ports devices this (hopefully) prevents them from being potentially misnumbered in the future and marks WAN port as wan.

> This does not rename the Linux network interfaces, is that correct?
Is the eth4 port in UCI now different from the eth4 interface in Linux? 

If I understood you correctly -- there's no difference between what's shown in the network config file and the output of ifconfig. The difference is between the port markings on the case and the OpenWrt port markings as I tried to explain just above.